src/app/components/ThemeToggle/ThemeToggle.module.scss

```diff
@@ -0,0 +1,20 @@
+@use '@/styles/variables.scss' as *;
+
+.switch {
+  $_icon: var(--icon, url(/theme-light.svg));
+  position: absolute;
+  top: 2rem;
+  right: $spacing-xl;
+  width: 2rem;
+  height: 2rem;
+  background: $_icon no-repeat center center / cover;
+  transition: $transition;
+
+  &_dark {
```

Comment:
`&--dark` it's a BEM modifier and not a BEM element so we need to use `--` to separate words

Reply:
Ok, but I assumed the element uses "__", the modifier uses "_".
https://en.bem.info/methodology/quick-start/#modifier